src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -39,31 +46,46 @@ module.exports.commands = [{
     tabUrl: function(tabName, relative) {
         return this.pageUrl(relative) + '/' + tabName;
     },
-    tabSelector: function(tabName) {
-        return 'nav.page-tabs a[href="' + this.tabUrl(tabName, true) + '"]';
-    },
     assertBasicLayoutOkay: function() {
-        this.waitForElementVisible(this.tabSelector('pipeline'));
-        this.waitForElementVisible(this.tabSelector('changes'));
-        this.waitForElementVisible(this.tabSelector('tests'));
-        this.waitForElementVisible(this.tabSelector('artifacts'));
+        this.waitForElementVisible(url.tabSelector('pipeline'));
+        this.waitForElementVisible(url.tabSelector('changes'));
+        this.waitForElementVisible(url.tabSelector('tests'));
+        this.waitForElementVisible(url.tabSelector('artifacts'));
         this.waitForElementVisible('@logHeader');
         // TODO: add class info to the page content so we can test it
         // Atm there's very little on the page that will allow us to test it.
         // E.g. nothing on the pipeline graph that allows us to find it.
     },
-    clickTab: function(browser, tab) {
+    assertTitle: function (expected) {
+        var self = this;
+        self.waitForElementVisible('@detailTitle');
+        self.getText('@detailTitle', function (response) {
+            self.assert.equal(typeof response, "object");
+            self.assert.equal(response.status, 0);
+            const urlProect = (response.value);
+            self.assert.equal(urlProect.indexOf(expected)>-1, true);
+            return self;
+        })
+
+    },
+    closeModal: function (browser) {
         var self = this;
-        const tabSelector = this.tabSelector(tab);
-        self.waitForElementVisible(tabSelector);
-        self.click(tabSelector);
+        self.waitForElementVisible('@closeButton');
+        self.click('@closeButton');
         browser.url(function (response) {
-                self.assert.equal(typeof response, "object");
-                self.assert.equal(response.status, 0);
-                // did we changed the url on  change?
-                self.assert.equal(response.value.includes(tab), true);
-                return self;
-            })
+            self.assert.equal(typeof response, "object");
+            self.assert.equal(response.status, 0);
+            this.pause(10000)
+            // FIXME JENKINS-36619 -> somehow the close in AT is not working as it should
+            // I debugged a bit and found out that the "previous" location is the same as
+            // current, this is the reason, why no url change is triggered. The question remains
+            // why that is happening
+            return self;
```

Comment:
Why the above `this.pause(10000)`. You know that kind of thing is notoriously flaky ... is there some connection with the comment that's after it? I don't follow.